ontology/observable/observable.ttl

```diff
@@ -10121,9 +10121,9 @@ observable:manuallyEnteredCount
 	.
 
 observable:manufacturer
-	a owl:DatatypeProperty ;
+	a owl:ObjectProperty ;
 	rdfs:label "manufacturer"@en ;
-	rdfs:range xsd:string ;
+	rdfs:range core:IdentityAbstraction ;
```

Comment:
core:IdentityAbstraction should only be used within the core namespace. It is a mechanism to allow references to Identity within core without causing cyclical importing.
In the observable namespace, uses such as this should directly use identity:Identity.

Reply:
Resolved in d7a593e.